File: Java/leetcode75/src/test/java/org/phoenix/leetcode/challenges/Problem60_MinCostClimbingStairsTest.java
package org.phoenix.leetcode.challenges;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class Problem60_MinCostClimbingStairsTest {

    private final Problem60_MinCostClimbingStairs test = new Problem60_MinCostClimbingStairs();

    @Test
    void minCostClimbingStairs() {
        int[] cost = new int[]{1, 100, 1, 1, 1, 100, 1, 1, 100, 1};

        assertEquals(6, test.minCostClimbingStairs(cost));
    }
}

File: JavaScript/leetcode75/src/test/Problem60_MinCostClimbingStairs.test.js
const minCostClimbingStairs = require("../main/Problem60_MinCostClimbingStairs");

test("should return min cost", () => {
  const cost = [1, 100, 1, 1, 1, 100, 1, 1, 100, 1];

  expect(minCostClimbingStairs(cost)).toBe(6);
});
